McCalib/src/geometrytools.cpp

```diff
@@ -537,11 +537,8 @@ cv::Mat handeyeBootstratpTranslationCalibration(
 
   // if enough sucess (at least 3) then compute median value
```

Comment:
Is the comment outdated ?

Reply:
Sorry, what do you mean and what do you want me to do here?

Reply:
Nothing special that seems good~